nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/java/org/apache/nifi/script/ScriptingComponentUtils.java

```diff
@@ -64,5 +68,15 @@
             .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
+
+    /** A property descriptor for specifying whether to use dynamic relationships or not */
+    public static final PropertyDescriptor USE_DYNAMIC_RELATIONSHIPS = new PropertyDescriptor.Builder()
+            .name("Use Dynamic Relationships")
+            .description("Whether properties prefixed with \"REL_\" should create a dynamic relationship or not. The default will change to \"true\" in NIFI 2.0.")
```

Comment:
Let's delete the reference to changing the default, in case we don't make the change in NiFi 2.0 we won't be providing false information in the doc. Changing the default should be transparent to the user, because any existing instances in a flow migrated to NiFi 2.0 will retain the setting whether it is the default or not. Only new instances will have a different default, and can be changed at that time by the user. It would probably be worth mentioning in any migration document when/if the default is changed, just for completeness, but the impact should be negligible.

Reply:
done